static/js/ui_init.js

```diff
@@ -90,7 +90,7 @@ exports.initialize_kitchen_sink_stuff = function () {
     // propagation in all cases.  Also, ignore the event if the
     // element is already at the top or bottom.  Otherwise we get a
     // new scroll event on the parent (?).
-    $('.modal-body, .scrolling_list, input, textarea').on('wheel', function (e) {
+    $('input, textarea').on('wheel', function (e) {
```

Comment:
What's the story with this change?

Reply:
With this code blocking wheel events, the scrollbars weren’t working at all in some cases. I optimistically assumed that the issue this code was trying to work around was likely specific to perfect-scrollbar, so I figured it’s worth trying to remove it under the Less Code Is Better principle. If the issue reappears, we’ll need to think about it more.

Reply:
I spelunked this a bit, and it's ca2b48e1ee754636a00490460493f1adb6d2dc04 and 45e515cc20fbeee73b91497fdd5005f2c06cde3b that introduce this.  

I think that predates perfect-scrollbar (1eae686443ed3f8c58156af6885a8ff9a4a683ac) by many months.  I would test scrolling in "keyboard shortcuts", the sidebars, and various settings views, and make sure they don't move the scroll position in the message feed; if that works, I'm happy to assume that the ancient bug that motivated this no longer exist.

(As a future note, it's best to give this sort of reasoning in a commit message or comments you add on the PR after opening it rather than just not mentioning it, since a reviewer is definitely going to ask why this random thing was removed).

Reply:
Upon further investigation, I think neither the current behavior nor my modified behavior were fully correct, so I’ve made the changes necessary to reimplement the current behavior for now.